src/data/posts.service.ts

```diff
@@ -54,6 +60,7 @@ export class PostsService {
       where: {
         parentId: IsNull(),
         id: idQuery,
+        ...creatorQuery
```

Comment:
```suggestion
        ...(creator ? {creator} : {})
```
this would achieve the same right?

Reply:
(without the code on lines 48-52)